.github/workflows/expo-go.yml

```diff
@@ -38,13 +38,13 @@ jobs:
       - name: 🧶 Yarn install
         if: steps.expo-caches.outputs.yarn-workspace-hit != 'true'
         run: yarn install --frozen-lockfile
-      - name: 🛠 Compile Home sources
+      - name: 🛠 Compile Expo Go sources
```

Comment:
i feel that `Expo Go` is not quite correct because it's only "home". how about use the name as "Expo Go Launcher"?